web/locales/spontaneous-speech/en/pages/contribution/code-switching.ftl

```diff
@@ -0,0 +1,50 @@
+## Specific to code-switching datasets
+
+## Errors
+cs-error-fetching-datasets = Something went wrong while fetching contributable datasets. Please try again later.
```

Comment:
Perhaps: *Something went wrong while fetching the list of available datasets. Please try again later.*

E.g. what is being fetched?